checkpoint_manager.json

```diff
@@ -0,0 +1,3 @@
+{
+
+}
```

Comment:
We decided to use 'yaml' everywhere.

Reply:
it could take time to change, json is in my opinion better solution for this logs/register system

Reply:
changed to yaml